relay-metrics/src/aggregation.rs

```diff
@@ -520,6 +520,21 @@ impl BucketValue {
             Self::Distribution(m) => m.internal_size(),
         }
     }
+
+    /// Estimates the number of bytes needed to encode the bucket.
+    /// Note that this does not necessarily match the exact memory footprint of the bucket,
+    /// because datastructures might have a memory overhead.
+    ///
+    /// This is very similar to [`relative_size`], which can possibly be removed.
+    pub fn cost(&self) -> usize {
```

Comment:
Not sure if `cost` is the best name for what we're modeling here. Open for suggestions.